### PR TITLE
willPaybyCheck now routes to corresponding confirmation page

### PR DESCRIPTION
**Pull request by claudianguyen**
willPaybyCheck determines which payment method to show on confirmation page.  Still need to make the changes in confirmation page to show the appropriate payment method (check or paypal)

### CODE REVIEW

File: frontend/jsx/common/utils/enum.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/** Donation related items */
```
Comment (ShahilZ) [inline frontend/jsx/common/utils/enum.js:1]: Nitpick: How about renaming to `enums.js`

File: frontend/jsx/pages/SponsorInfoPage.jsx [hunk 2]
```diff
@@ -32,34 +34,13 @@ class SponsorInfoPage extends Component {
     constructor(props) {
         super(props);
         this.state = {
-           
-            donation: {
```
Comment (ShahilZ) [inline frontend/jsx/pages/SponsorInfoPage.jsx:36]: TODO: Transform into functional component! At least, as long as it does not require state.

**Review by ShahilZ — APPROVED** [review]
Nice work! Just a couple of small comments

File: frontend/jsx/common/utils/enum.js (hunk 1 above)
Comment (claudianguyen) [reply frontend/jsx/common/utils/enum.js]: Sure thing

File: frontend/jsx/pages/SponsorInfoPage.jsx (hunk 2 above)
Comment (claudianguyen) [reply frontend/jsx/pages/SponsorInfoPage.jsx:36]: Right, good call

**Pull request merged**